{"new":null,"update":[{"id":613526720,"node_id":"R_kgDOJJGswA","name":"CVE-2021-45010","full_name":"Syd-SydneyJr/CVE-2021-45010","private":false,"owner":{"login":"Syd-SydneyJr","id":127783310,"node_id":"U_kgDOB53Rjg","avatar_url":"https://avatars.githubusercontent.com/u/127783310?v=4","url":"https://api.github.com/users/Syd-SydneyJr","html_url":"https://github.com/Syd-SydneyJr","type":"User","site_admin":false},"html_url":"https://github.com/Syd-SydneyJr/CVE-2021-45010","description":"","fork":false,"url":"https://api.github.com/repos/Syd-SydneyJr/CVE-2021-45010","created_at":"2023-03-13T18:40:02Z","updated_at":"2023-03-14T00:32:44Z","pushed_at":"2023-03-13T18:41:41Z","stargazers_count":1,"watchers_count":1,"forks_count":0,"archived":false,"disabled":false,"allow_forking":true,"is_template":false,"topics":[],"visibility":"public","forks":0,"watchers":1,"default_branch":"main","score":1},{"id":592012076,"node_id":"R_kgDOI0ljLA","name":"Python-exploit-CVE-2020-25213","full_name":"BLY-Coder/Python-exploit-CVE-2020-25213","private":false,"owner":{"login":"BLY-Coder","id":63339164,"node_id":"MDQ6VXNlcjYzMzM5MTY0","avatar_url":"https://avatars.githubusercontent.com/u/63339164?v=4","url":"https://api.github.com/users/BLY-Coder","html_url":"https://github.com/BLY-Coder","type":"User","site_admin":false},"html_url":"https://github.com/BLY-Coder/Python-exploit-CVE-2020-25213","description":"Python exploit for RCE in Wordpress","fork":false,"url":"https://api.github.com/repos/BLY-Coder/Python-exploit-CVE-2020-25213","created_at":"2023-01-22T16:54:25Z","updated_at":"2023-03-14T00:31:16Z","pushed_at":"2023-03-13T13:32:50Z","stargazers_count":1,"watchers_count":1,"forks_count":2,"archived":false,"disabled":false,"allow_forking":true,"is_template":false,"topics":[],"visibility":"public","forks":2,"watchers":1,"default_branch":"main","score":1}]}